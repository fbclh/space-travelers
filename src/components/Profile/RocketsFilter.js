export default function RocketsFilter() {
  return <div>RocketsFilter</div>;
}
