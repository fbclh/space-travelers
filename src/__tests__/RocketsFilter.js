import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import RocketsFilter from '../components/Profile/RocketsFilter';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <RocketsFilter />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
