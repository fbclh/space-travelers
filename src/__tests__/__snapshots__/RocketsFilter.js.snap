// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="reserved-rockets"
>
  <h1>
    My Rockets
  </h1>
  <div
    className="reserved-list"
  />
</div>
`;
